tests/integration/test_fim/test_files/test_max_eps/test_max_eps_synchronization.py

```diff
@@ -63,92 +63,115 @@
+pytestmark = [TIER1, AGENT, WINDOWS, LINUX]
 
 # Variables
-test_directories_no_delete = [os.path.join(PREFIX, 'testdir1')]
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
 
-directory_str = ','.join(test_directories_no_delete)
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), DATA)
+configurations_path = os.path.join(test_data_path, YAML_CONF_MAX_EPS_SYNC)
+
+test_directories = [os.path.join(PREFIX, TEST_DIR_1)]
+conf_params = {TEST_DIRECTORIES: test_directories[0]}
+
+ERR_MSG_MULTIPLE_FILES_CREATION = 'Multiple files could not be created.'
 
-wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
-test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
-configurations_path = os.path.join(test_data_path, 'wazuh_conf_synchro.yaml')
-testdir1 = os.path.join(PREFIX, 'testdir1')
 
 # Configurations
-conf_params = {'TEST_DIRECTORIES': directory_str,
-               'MODULE_NAME': __name__}
 
-eps_values = ['50', '10']
-test_modes = ['realtime'] if sys.platform == 'linux' or sys.platform == 'win32' else ['scheduled']
+# Test with the minimum, and the default value
+eps_values = ['1', '100']
 
-p, m = generate_params(extra_params=conf_params, apply_to_all=({'MAX_EPS': eps_value} for eps_value in eps_values),
-                       modes=test_modes)
-configurations = load_wazuh_configurations(configurations_path, __name__, params=p, metadata=m)
+parameters, metadata = generate_params(extra_params=conf_params,
+                                       modes=['scheduled', 'realtime', 'whodata'],
+                                       apply_to_all=({'MAX_EPS': eps_value} for eps_value in eps_values))
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f"{x['fim_mode']}_mode_{x['max_eps']}_max_eps" for x in metadata]
 
+local_internal_options = FIM_DEFAULT_LOCAL_INTERNAL_OPTIONS
 
 # Fixtures
 
-@pytest.fixture(scope='module', params=configurations)
+@pytest.fixture(scope='module', params=configurations, ids=configuration_ids)
 def get_configuration(request):
     """Get configurations from the module."""
     return request.param
 
 
 @pytest.fixture(scope='module')
-def create_files(get_configuration):
+def create_multiple_files(get_configuration):
+    """Create multiple files of a spectific type."""
     max_eps = get_configuration['metadata']['max_eps']
     mode = get_configuration['metadata']['fim_mode']
-    for i in range(int(max_eps) * 5):
-        create_file(REGULAR, testdir1, f'test{i}_{mode}_{max_eps}', content='')
+    os.makedirs(test_directories[0], exist_ok=True, mode=0o777)
+    try:
+        for i in range(int(max_eps) + 5):
+            file_name = f'file{i}_to_max_eps_{max_eps}_{mode}_mode'
+            create_regular_file(test_directories[0], file_name, content='')
+    except OSError:
+        logger.info(ERR_MSG_MULTIPLE_FILES_CREATION)
 
 
 @pytest.fixture(scope='function')
```

Comment:
This function already exists in deps/wazuh_testing/wazuh_testing/tools/file.py called delete_path_recursively, please use this function. But If you want to use a fixture for a reason, invoke the function inside the fixture